src/test/java/uk/gov/hmcts/reform/sscs/model/HearingDurationTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.BenefitCode.UC;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.Issue.US;
+
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import uk.gov.hmcts.reform.sscs.ccd.domain.BenefitCode;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Issue;
+import uk.gov.hmcts.reform.sscs.service.HearingDurationsService;
+
+public class HearingDurationTest {
+
+    HearingDurationsService hearingDurations;
+
+    @Before
+    public void setup() {
+        hearingDurations = new HearingDurationsService();
+    }
+
+    @DisplayName("When valid Benefit Code and Issue Code is given to getHearingDuration the valid mapping is returned")
+    @Test
+    public void getHearingDuration() {
+        HearingDuration result = hearingDurations.getHearingDuration("003", "LE");
+
+        assertEquals(BenefitCode.PIP_REASSESSMENT_CASE, result.getBenefitCode());
+        assertThat(result.getIssue()).isEqualTo(Issue.LE);
+        assertThat(result.getDurationFaceToFace()).isEqualTo(60);
+        assertThat(result.getDurationInterpreter()).isEqualTo(90);
+        assertThat(result.getDurationPaper()).isEqualTo(30);
+    }
```

Comment:
Better to have one test for each assert so when the first one fails, it doesn't hide the result of others.
I.e. this would only show an error on the first failing assert, so you sort that then you have to run it again to see the next assert fail, etc...